src/sentry/culprit.py

```diff
@@ -69,3 +72,11 @@ def get_frame_culprit(frame, platform):
         # to a unicode string if needed.
         return "{}({})".format(frame.get("function") or "?", fileloc)
     return "{} in {}".format(fileloc, frame.get("function") or "?")
+
+
+def get_nel_culprit(data_nel):
+    body = data_nel.get("body")
+    ty = body.get("type", "<missing>")
+    if ty == "http.error":
+        return NEL_CULPRITS[ty].format(body.get("status_code"))
+    return NEL_CULPRITS.get(ty, f"Unknown type: {ty}")
```

Comment:
This might be confusing, because it is not clear that "Unknown type" comes from a sentry-side lookup failure and not the browser. How about we fall back to `ty`, i.e.
```suggestion
    return NEL_CULPRITS.get(ty, ty)
```